packages/docsearch-react/src/utils/unescape.ts

```diff
@@ -0,0 +1,8 @@
+const regexEscapedHtml = /(<mark>|<\/mark>)/g;
+const regexHasEscapedHtml = RegExp(regexEscapedHtml.source);
+
+export function unescape(value: string): string {
```

Comment:
Just realized that `unescape` can be confusing, maybe `removeHighlightTags`?

Reply:
renamed in https://github.com/algolia/docsearch/pull/1001/commits/b56836259d96aeabdcc4a46a59bb3ea9030daef9